CHANGELOG.rst

```diff
@@ -14,6 +14,19 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[2.6.1] - 2021-01-25
+~~~~~~~~~~~~~~~~~~~~~
+* Add a dropdown component.
+* If the "data-enable-exam-resume-proctoring-improvements" data attribute on the element of the ProctoredExamAttemptView
+  Backbone is true, use the dropdown menu component on the Instructor Dashboard Proctored Exam Attempt panel for proctored exam attempts in the error state.
+* The dropdown provides two options: Resume and Reset.
+* Resume transitions the exam attempt into the ready_to_resume state.
+* Reset  behaves the same as the previous reset functionality, originally exposed via the [x] link.
+* If the "data-enable-exam-resume-proctoring-improvements" data attribute on the element of the ProctoredExamAttemptView
+  Backbone is true, change the [x] link to Reset for exam attempts in other states.
```

Comment:
#NIT I feel like line 25 and 26 repeats what you already said in line 20-24. So, these can be removed as dupe?